### PR TITLE
debtor(adjacent) changes- phobia changes, psychotic brawling rework

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
re-pulls #4306 with some changes

-the debtor does not lose their phobia when removing their hat
-phobias are far, far less punishing for the player. they no longer instantly snowball into fainting, and it's easier to lose built-up stress
-reworks psychotic brawling. this is overall a nerf, and wont be an instant win with a meteor punch anymore, but also gives the debtor a degree of control over what they do:
Disarm intent can
*cause the debtor to knock down the target for a decent time, and launch them a long distance (1/6 chance)
*cause the debtor to knock down the target for a very short time and launch them a short distance, or paralyze them if they are already down (1/6 chance)
*cause the debtor to attack as if on harm intent (1/6 chance)
*cause the debtor to steal an item, knock the target down, or stun them if they are already down(1/6 chance)
*cause the debtor to stumble over themselves(2/6 chance)
Grab intent can
*make the debtor hug targets and lose their grabs(2/6 chance)
*do a normal grab(2/6 chance)
*do an aggro grab(1/6 chance)
*effectively but not actually stun a debtor for 1.5 seconds, after which they throw themselves at the target, with an instant neck grab if they hit(1/6 chance)
If the target of a grab is not conscious, the debtor will instead do a normal grab, so changeling debtors can still absorb targets
Harm intent can
*do the old meteor punch, but with knockdown instead of stunning(1/6 chance)
*bite, which transmits diseases between the debtor and the target, does low damage, and can rarely cause the target to get a new disease(1/6 chance)
*poke, which has very low attack delay but next to no damage(1/6 chance)
*cry, which stuns the debtor(1/6 chance)
*headbutt, stunning the debtor and the target (1/6 chance)
*do a normal hit (1/6 chance)

## Why It's Good For The Game
makes the Debtor job a bit less shit, and makes phobias far less debilitating

## Changelog
:cl:
tweak: debtors no longer lose their conspiracy phobia when removing their hats
tweak: phobia has been nerfed somewhat, so it should deteriorate faster and be less likely to instantly sleep you
balance: psychotic brawling has been reworked and should be a bit more balanced for both the target and the brawler
/:cl:

### CODE REVIEW

**Comment by St0rmC4st3r** [comment]
While you are at it, maybe make lizard debtors not scared of lizards? It doesn't make much sense and is really annoying.

File: code/datums/martial/psychotic_brawl.dm [hunk 1]
```diff
@@ -3,69 +3,175 @@
...
-	return psycho_attack(A,D)
+	return psycho_disarm(A,D)
 
 /datum/martial_art/psychotic_brawling/grab_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
-	if(HAS_TRAIT(A, TRAIT_PACIFISM))
-		return FALSE
-	return psycho_attack(A,D)
+	return psycho_grab(A,D)
 
 /datum/martial_art/psychotic_brawling/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
```
Comment (ike709) [inline code/datums/martial/psychotic_brawl.dm:11]: Shouldn't the `harm_act()` check for pacifism since it can actually do damage?

**Comment by ike709** [comment]
Maintainers have decided to just remove debtor. If you want the phobia changes reviewed, re-PR it without the debtor stuff.

File: code/datums/martial/psychotic_brawl.dm (hunk 1 above)
Comment (zeskorion) [reply code/datums/martial/psychotic_brawl.dm:11]: Harm_act checks for pacifism by default

**Pull request closed without merging**